bindata/network/ovn-kubernetes/common/ipsec-containerized.yaml

```diff
@@ -90,11 +96,6 @@ spec:
           export KUBECONFIG=/var/run/ovnkube-kubeconfig
 {{ end }}
 
-          if rpm --dbpath=/usr/share/rpm -q libreswan; then
-            echo "host has libreswan and therefore ipsec will be configured by ipsec daemonset, this ovn ipsec container doesnt need to init anything"
-            exit 0
-          fi
-
```

Comment:
I think you need this with similar condition to the node selector
on hcp, you dont want the containerized version to deploy on RHCOS workers.. (and you are not using the node selector there..)

Reply:
you'll have to also solve the rpm db location issue
might not be able to escape it by using 2 different daemon sets

Reply:
The  rpm db location is used for checking libreswan package is present or not. But irrespective of we render both ipsec daemon sets, finally there would  be either ipsec-containerized or ipsec-host pod being scheduled on a node and libreswan packages are expected to be present for pod to run.  so these checks are no longer needed, hence removed it.
for hcp clusters, we support only containerized version even on rhcos because machine configs are not supported there.